src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,47 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post,
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    id,
+    title,
+    body,
+    comments,
+    user,
+  } = post;
+
+  return (
+    <div
+      className="PostInfo"
+      key={id}
+    >
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        {user ? (<UserInfo user={user} />) : ''}
```

Comment:
You can just do like that
```suggestion
        {user && (<UserInfo user={user} />)}
```